app/javascript/helpers/local-storage.js

```diff
@@ -0,0 +1,26 @@
+function get(map, key, values) {
```

Comment:
Function 'get' has too many statements (12). Maximum allowed is 10  max-statements